Enhanced_Zeus_Modules.sqf

```diff
@@ -9359,48 +9359,102 @@ MAZ_EZM_fnc_initFunction = {
-					private _camSrc90 = [(_target select 0) + _circleRadius, _target select 1, (_target select 2) + _camHeight];
-					
-					private _camera = "camera" camCreate _camSrc0;
-					_camera cameraEffect ["internal", "back"];
-					_camera camPrepareTarget _camTarget;
-					_camera camSetFov 1;
-					_camera camCommitPrepared 0;
-
-					_camera camPreparePos _camSrc90;
-					_camera camCommitPrepared 15;
-					waitUntil { camCommitted _camera };
-					cutRsc ["RscStatic", "PLAIN"];
-					sleep 0.4;
-
-					_camera cameraEffect ["terminate", "back"];
-					camDestroy _camera;
-
-					comment "Remove color correction right after cutscene is done so we don't have to remoteExec it";
-					ppEffectDestroy HYPER_PP_CC_Cinematic;
-
-					comment "Re-enable simulation on player vehicles";
-					if(player != vehicle player) then {
-						[vehicle player, true] remoteExec ["enableSimulationGlobal", 2];
+			switch (_cinematicType) do {
+				case "Flyby": {
+					comment "in flyby mode, we select the module location as our target, and the camera paths are automatically designated at 0 and 90 degrees";
+					HYPER_fnc_remoteCamera = {
+						params ["_target"];
+						private _zeus = !isNull (getAssignedCuratorLogic player);
+						private _camTarget = "Land_HelipadEmpty_F" createVehicleLocal _target;
+						private _circleRadius = 200;
+						private _camHeight = 200;
+						private _camSrc0 = [_target select 0, (_target select 1) + _circleRadius, (_target select 2) + _camHeight];
+						private _camSrc90 = [(_target select 0) + _circleRadius, _target select 1, (_target select 2) + _camHeight];
+						
+						private _camera = "camera" camCreate _camSrc0;
+						_camera cameraEffect ["internal", "back"];
+						_camera camPrepareTarget _camTarget;
+						_camera camSetFov 1;
+						_camera camCommitPrepared 0;
+
+						_camera camPreparePos _camSrc90;
+						_camera camCommitPrepared 15;
+						waitUntil { camCommitted _camera };
+						cutRsc ["RscStatic", "PLAIN"];
+						sleep 0.4;
+
+						_camera cameraEffect ["terminate", "back"];
+						camDestroy _camera;
+
+						ppEffectDestroy HYPER_PP_CC_Cinematic;
+						
+						cutText ["", "BLACK IN", 2];
+						[1, 2, true, true] call BIS_fnc_cinemaBorder;
+
+						comment "Re-enable simulation on player vehicles";
+						if(player != vehicle player) then {
+							[vehicle player, true] remoteExec ["enableSimulationGlobal", 2];
+						};
+
+						if(_zeus) then {
+							sleep 0.2;
+							openCuratorInterface;
+						};
 					};
+					[[_target],HYPER_fnc_remoteCamera] remoteExec ["spawn", _allPlayers];
+				};
+				case "News": {
+					[[],{
+						playSoundUI ["a3\sounds_f\vehicles\air\heli_attack_01\heli_attack_01_ext_rotor.wss", 0.2];
+						sleep 7.5;
+						playSoundUI ["a3\sounds_f\vehicles\air\heli_attack_01\heli_attack_01_ext_rotor.wss", 0.2];
+					}] remoteExec ["spawn", _allPlayers];
+					private _AANParams = [
+						parseText _briefingName,
```

Comment:
The text utilizes [Structured Text](https://community.bistudio.com/wiki/Structured_Text). Make the text size 2:
`"<t size='2'> [TEXT HERE] </t>"`

Reply:
Specifically for the headline, not the rolling text.